docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,194 @@
+* Allow field name declaration in ROW literal.  For example, `row(1 as a, 2 as b)` is now legal. ({issue}`25261`)
+* Deprecate `EXPLAIN (TYPE LOGICAL)`. `EXPLAIN (TYPE DISTRIBUTED)` should be used instead. ({issue}`27434`)
+* Remove `prefer_streaming_operators` session property. `SET SESSION task_concurrency=1` can be used to achieve the same behaviour as `prefer_streaming_operators`. ({issue}`27506`)
+* Improve performance of remote data exchanges on CPUs supporting the required SIMD extensions. This can be disabled by setting `exchange.experimental.vectorized-serde.enabled=false`. ({issue}`27426`, {issue}`26919`)
+* Enhance after `RemoveTrivialFilters` call `RemoveEmptyUnionBranches` to prune empty union branches. ({issue}`21506`)
+* Improve performance of {func}`array_sort` function. ({issue}`27272`)
+* Improve performance of {func}`repeat` function. ({issue}`27369`)
+* Improve performance of data exchanges involving variable width data. ({issue}`27377`)
+* Fix planning failure of certain queries where part of the plan is optimized to empty values. ({issue}`21506`)
+* Fix configuring partitioned layout for spooling protocol. ({issue}`27247`)
+* Fix `EXPLAIN (TYPE IO)` failure when query constraint contains type which cannot be cast to `varchar`. ({issue}`27433`)
+
+## Security
+
+## Web UI
+
+* Fix preview UI to render single line queries. ({issue}`27328`)
+* Fix preview UI to render catalog properties correctly. ({issue}`27327`)
+
+## JDBC driver
+
+* Add a `--extra-header` option to support sending arbitrary HTTP headers. ({issue}`15826`)
+
+## Docker image
+
+* Update JDK to 25.0.1. ({issue}`27117`)
+
+## CLI
+
+* Add a `--extra-header` option to support sending arbitrary HTTP headers. ({issue}`15826`)
+* Fix TLS connection failures when using unqualified (single-label, without ".") hostnames. Default Java TLS does not send Server Name Indication (SNI) for such hostnames, which could cause certificate mismatches during TLS termination. ({issue} `27478`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+* Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. The new property will be removed in a future version.
+
+## Delta Lake connector
+
+* Add `APPLICATION_DEFAULT` authentication type for GCS. ({issue}`26984`)
+* Remove unauthentication access support in `SERVICE_ACCOUNT`. ({issue}`26984`)
+* Rename `s3.exclusive-create` config to `delta.s3.transaction-log-conditional-writes.enabled`. ({issue}`27372`)
+* Fix incorrect results due to incorrect pushdown of `IS NOT DISTINCT FROM`. ({issue}`27213`)
+* Fix failure when writing to tables written by Databricks 17.3. ({issue}`27100`)
+* Harden hierarchical namespace check in Azure with root blob fallback check. ({issue}`27278`)
+* Avoid reading unusually large pages from parquet files. ({issue}`27148`)
```

Comment:
What's the user visible impact?

Reply:
It fails queries attempting to read parquet files with massive pages, as doing so risks worker OOMs. This is rare but could happen with columns containing massive strings or binary data.
Maybe reword it to `Avoid worker crashes by failing queries attempting to read columns with huge values in parquet files`